### PR TITLE
Added drawBitmap

### PR DESCRIPTION
**Pull request by koen1711**
**Didn't compile it for arduino_compressed!**

### CODE REVIEW

**Comment by koen1711** [comment]
Didn't compile the stuff I added.

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,5 @@ chromedriver
 build/
 dist/
 temp/
+yarn.lock
+yarn.lock
```
Comment (paulsiersma) [inline .gitignore:24]: yarn.lock is added twice.

File: demos/code/index.html [hunk 2]
```diff
@@ -432,10 +452,8 @@ <h1><a href="https://developers.google.com/blockly/">Blockly</a>&rlm; &gt;
             <field name="TEXT">tekst</field>
           </shadow>
         </value>
-        <value name="VALUE">
-          <shadow type="math_number">
-            <field name="NUM">0</field>
-          </shadow>
+
```
Comment (paulsiersma) [inline demos/code/index.html:455]: I think removing this was a mistake, please revert

File: generators/arduino/leaphy_extra.js [hunk 3]
```diff
@@ -9,10 +9,44 @@
  */
  'use strict';
 
+
+
 goog.module('Blockly.Arduino.leaphyExtra');
  
 const { arduinoGenerator: Arduino } = goog.require('Blockly.Arduino');
 
+//function GenerateBitmap(path) {
```
Comment (paulsiersma) [inline generators/arduino/leaphy_extra.js:18]: Please remove commented code

File: msg/messages.js [hunk 4]
```diff
@@ -243,6 +243,8 @@ Blockly.Msg.LEAPHY_DISPLAY_CLEAR = 'Clear display';
 Blockly.Msg.LEAPHY_DISPLAY_PRINT = 'Display - Set Ln.';
 /** @type {string} */
 /// Print display
+Blockly.Msg.LEAPHY_DRAW_BITMAP = 'Display %1 %2 %3 %4';
+
 Blockly.Msg.LEAPHY_DISPLAY_DISPLAY = 'Show on display';
```
Comment (paulsiersma) [inline msg/messages.js]:
```suggestion
/** @type {string} */
/// Draw Bitmap
Blockly.Msg.LEAPHY_DRAW_BITMAP = 'Display %1 %2 %3 %4';
/** @type {string} */
/// Print display
Blockly.Msg.LEAPHY_DISPLAY_DISPLAY = 'Show on display';
```

**Review by paulsiersma — CHANGES_REQUESTED** [review]
A couple of things to check.

**Pull request closed without merging**